raycast/BaseGame.py

```diff
@@ -0,0 +1,145 @@
+import collections
+import time
+from abc import ABCMeta
+
+import katagames_sdk.engine as kataen
+
+
+pygame = kataen.import_pygame()
+EventReceiver = kataen.EventReceiver
+EngineEvTypes = kataen.EngineEvTypes
+
+
+class BaseGame(metaclass=ABCMeta):
+    """
+    Base class for games, this is staged for the katasdk0.0.7 release
+    """
+
+    FPS_TRACKING_DEFAULT_SS = 13
+
+    def __init__(self, track_fps=True):
```

Comment:
`track_fps` should maybe just always be enabled; it adds close to zero overhead I'm guessing.

Reply:
There is a performance issue for the web bundle and I have not located the bottleneck yet so let's keep the switch and I disable the fps tracking by default.
I need to stay cautious. For example the performance of the '.popleft()'  operation for the 'collections.deque' object is unknown when it is running through Brython